mlflow/server/js/src/shared/building_blocks/PageHeader.jsx

```diff
@@ -69,9 +77,9 @@ export class PageHeader extends React.Component {
               {feedbackLink}
             </>
           }
-          />
+        />
         <Spacer />
       </>
     );
   }
-}
+}
```

Comment:
Looks like this file is not linted.

Reply:
`eslint src --fix` only fixes `.js` files?

Reply:
```diff
> git diff
diff --git a/mlflow/server/js/package.json b/mlflow/server/js/package.json
index 348ac30c1..220a95448 100644
--- a/mlflow/server/js/package.json
+++ b/mlflow/server/js/package.json
@@ -136,7 +136,7 @@
     "extract-i18n:base": "node scripts/extract-i18n.js",
     "extract-i18n:check": "yarn extract-i18n:base --lint",
     "extract-i18n": "yarn extract-i18n:base",
-    "lint": "eslint src",
+    "lint": "eslint src --ext .js --ext .jsx",
```

```
> yarn lint
jsxBracketSameLine is deprecated.

/home/haru/Desktop/repositories/mlflow/mlflow/server/js/src/shared/building_blocks/PageHeader.jsx
  61:9  error  'feedbackLink' is never reassigned. Use 'const' instead  prefer-const
  85:2  error  Insert `⏎`                                               prettier/prettier
  85:2  error  Newline required at end of file but not found            eol-last

✖ 3 problems (3 errors, 0 warnings)
  3 errors and 0 warnings potentially fixable with the `--fix` option.
```

`--ext .jsx` enables lint check on `.jsx` files.

Reply:
Ahhh good catch. Maybe we should do one for tsx as well since we have enabled TypeScript. cc @hubertzub-db

Reply:
Adding `.tsx` sounds good. Do we have `.ts` files?

Reply:
Yes! So now with fixlinting js/jsx/ts/tsx we should be covered, thanks!